llvm/lib/Target/RISCV/RISCVTargetMachine.cpp

```diff
@@ -89,6 +90,10 @@ static cl::opt<bool>
                            cl::desc("Enable the loop data prefetch pass"),
                            cl::init(true));
 
+static cl::opt<bool> EnableSplitRegAlloc("riscv-split-regalloc", cl::Hidden,
+                                   cl::desc("Enable Split RegisterAlloc for RVV"),
+                                   cl::init(false));
+
```

Comment:
Should we just enable split register allocation by default? I didn't notice any regressions with this enabled on the [llvm test suite](https://github.com/llvm/llvm-project/files/13332331/regalloc-split.patch), and it doesn't seem to affect the output at all on RV64g (i.e. without v)

Reply:
Thanks for reviewing.

I think the splitting register allocation only gain benefit when the following optimization is ready. Perhaps we can enable it by default when the following optimizations are stable.